File: UETshare/src/main/java/com/example/uetshare/response/dto/CommentDto.java
package com.example.uetshare.response.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.util.Calendar;

@Setter
@Getter
@AllArgsConstructor
@NoArgsConstructor
public class CommentDto {

    private Long id;
    private String content;
    private String image;
    private Long question_id;
    private Long account_id;
    private Calendar time;
    private AccountDto accountDto;
    private Integer like_quantity;
    private Integer sub_comment_quantity;
    private Boolean liked;
}

File: app/src/main/java/ie/app/uetstudents/Entity/Comment/get/CommentDto.kt
package ie.app.uetstudents.Entity.Comment.get

import ie.app.uetstudents.data.response.AccountDto

data class CommentDto(
    val account_id: Int? = null,
    val content: String? = null,
    val id: Int? = null,
    val image: String? = null,
    val question_id: Int? = null,
    val time: String? = null,
    val like_quantity : Int = 0,
    val sub_comment_quantity : Int = 0,
    var liked : Boolean = false,
    val accountDto : AccountDto? = null
)